pkg/util/net.go

```diff
@@ -11,13 +12,13 @@ import (
 
 // GetFirstAddressOf returns the first IPv4 address of the supplied interface names.
 func GetFirstAddressOf(names []string) (string, error) {
```

Comment:
Let's also update the comment for this function and mention handling IP Addresses in APIPA range.

Reply:
Sounds good, updated.